### DISCUSSION TITLE
nginx reverse proxy with iOS18

### DISCUSSION THREAD

**Discussion by ajabir** [nginx/nginx#470] [Q&A]
Has anyone encountered compatibility issue with nginx reverse proxy and latest iOS18?  We have discovered that our iOS18 users are not able to authenticate with our self-developed web application sitting behind nginx reverse proxy. The same webapp was working fine with previous iOS and clients from other platforms.  The app also works fine if iOS18 device connect directly to the app port, bypassing the proxy.

**Comment by princessfrombabylon99** [comment]
@ajabir I read some similar topic in some forum. It was mentioned , your app might be using API's with IP's , you gotta try replacing IP with domain name for API's

**Comment by Maryna-f5** [comment]
Moving this issue to Discussions
